apps/roam/src/utils/getDiscourseNodeType.ts

```diff
@@ -0,0 +1,48 @@
+import getPageTitleByPageUid from "roamjs-components/queries/getPageTitleByPageUid";
+import getDiscourseNodes, { type DiscourseNode } from "./getDiscourseNodes";
+import "core-js/proposals/regexp-escaping";
+import format from "date-fns/esm/fp/format/index.js";
+
+// eslint-disable-next-line
```

Comment:
Why is this necessary?  Using `any` should be avoided at all costs.

Reply:
It was for that code, but that code is gone.